websites/W/watch.lonelil.com/presence.ts

```diff
@@ -0,0 +1,162 @@
+const presence = new Presence({
+		clientId: "1147910465134002276",
+	}),
+	getStrings = async () => {
+		return presence.getStrings(
+			{
+				play: "general.playing",
+				pause: "general.paused",
+				viewAMovie: "general.viewAMovie",
+				viewAShow: "general.viewAShow",
+				viewAnime: "general.viewAnime",
+				watchingMovie: "general.watchingMovie",
+				watchingShow: "general.watchingShow",
+				watchingLive: "general.watchingLive",
+				viewingHomePage: "general.viewHome",
+				loading: "kahoot.loadingPage",
```

Comment:
This kahoot string is probably not even loaded... I think you would need to add the kahoot strings in the multilang definition

Reply:
The Kahoot! presence and others don’t seem to have that though?

Reply:
https://docs.premid.app/en/dev/presence/metadata#multilanguage
True loads the general file and <service>.json